content/en/docs/languages/go/instrumentation.md

```diff
@@ -876,7 +875,160 @@ meterProvider := metric.NewMeterProvider(
 
 ## Logs
 
-The logs API is currently unstable, documentation TBA.
+Logs are distinct from metrics and traces in that **there is no user-facing
+OpenTelemetry logs API**. Instead, there is tooling to bridge logs from existing
+popular log packages (e.g. slog, logrus, zap, logr) into the OpenTelemetry
```

Comment:
```suggestion
popular log packages (such as slog, logrus, zap, logr) into the OpenTelemetry
```

Reply:
Same is in Java. Should I fix it there as well?

Reply:
yes, if you want to, but in a separate PR.